cmd/minikube/cmd/version.go

```diff
@@ -43,13 +48,57 @@ var versionCmd = &cobra.Command{
 			"minikubeVersion": minikubeVersion,
 			"commit":          gitCommitID,
 		}
+
+		if listPackagesVersions {
+			co := mustload.Running(ClusterFlagValue())
+			runner := co.CP.Runner
+			// docker version --format='{{.Client.Version}}'
+			// sudo crictl version
+			// runc --version
+			// crio version
+			// buildctl --version
+			// sudo ctr version
+			// sudo podman version
+
+			versionCMDS := map[string]*exec.Cmd{
+				"docker":     exec.Command("docker", "version", "--format={{.Client.Version}}"),
+				"containerd": exec.Command("containerd", "--version"),
+				"crio":       exec.Command("crio", "version"),
+				"podman":     exec.Command("sudo", "podman", "version"),
+				"crictl":     exec.Command("sudo", "crictl", "version"),
+				"buildctl":   exec.Command("buildctl", "--version"),
+				"ctr":        exec.Command("sudo", "ctr", "version"),
+				"runc":       exec.Command("runc", "--version"),
+			}
+			for k, v := range versionCMDS {
+				rr, err := runner.RunCmd(v)
+				if err != nil {
+					klog.Warningf("error getting %s's version: %v", k, err)
+					data[k] = "error"
```

Comment:
would be good to have more details here. Getting version may fail due to multiple reasons: non-zero exit code, permission error, tool is not installed, etc

Reply:
the error is display in the verbose logs (klog) but for UI it is better to not pollute with lines of Errors in an already messy blocks of versions, will be hard to figure out what is what.